tower/CHANGELOG.md

```diff
@@ -6,17 +6,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
+None.
+
+# 0.4.5 (February 10, 2021)
+
+### Added
+
 - **util**: Add `ServiceExt::map_future`. ([#542])
-- **util**: Make combinators implement `Debug` in more cases
-- `tracing` is now only pulled in for the features that need it.
 - **util**: Add `option_layer` to convert an `Option<Layer>` into a `Layer`. ([#555])
 - **builder**: Add `ServiceBuilder::option_layer` to optionally add a layer. ([#555])
-- **steer**: `Steer` now implements `Clone
 - **make**: Added `Shared` which lets you implement `MakeService` by cloning a
-  service.
+  service. ([#533])
+
+### Fixed
+
+- **util**: Make combinators implement `Debug` in more cases. ([#552])
```

Comment:
nit: can we clarify "in more cases"? specifically, I think this is that the `Debug` impl doesn't require the closure/function to be `Debug`...and in practice, i think these types will _never_ implement `Debug`....

Reply:
👍